vendor/dario.cat/mergo/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
```

Comment:
This should belong to `Update autogenerated` commit

Reply:
`Update dependencies` encompasses `go.mod`, `go.sum` and `vendor/` so it seems placed correctly